Backend/dev/migrations/rider-dashboard/0006-access-matrix-table-altered.sql

```diff
@@ -0,0 +1,195 @@
+CREATE OR REPLACE FUNCTION atlas_bap_dashboard.uuid_generate_v4() RETURNS character (36) AS $uuid_generate_v4$
+    BEGIN
+        RETURN (uuid_in((md5((random())::text))::cstring));
+    END;
+$uuid_generate_v4$ LANGUAGE plpgsql;
+
+ALTER TABLE atlas_bap_dashboard.access_matrix ADD COLUMN user_action_type varchar (255);
+ALTER TABLE atlas_bap_dashboard.access_matrix DROP CONSTRAINT unique_role_id_api_entity;
+ALTER TABLE atlas_bap_dashboard.access_matrix ADD CONSTRAINT unique_role_id_api_entity_user_action_type UNIQUE (role_id, api_entity, user_action_type);
+
+INSERT INTO atlas_bap_dashboard.access_matrix (id, role_id, api_entity, user_access_type, user_action_type)
+    (
+    SELECT
+        atlas_bap_dashboard.uuid_generate_v4(),
+        T1.role_id,
+        T1.api_entity,
+        'USER_FULL_ACCESS',
+        unnest (ARRAY ['MERCHANT_UPDATE','MERCHANT_COMMON_CONFIG_UPDATE','DRIVER_POOL_CONFIG_UPDATE','DRIVER_POOL_CONFIG_CREATE','DRIVER_INTELLIGENT_POOL_CONFIG_UPDATE','ONBOARDING_DOCUMENT_CONFIG_UPDATE','ONBOARDING_DOCUMENT_CONFIG_CREATE' ,'MAPS_SERVICE_CONFIG_UPDATE', 'MAPS_SERVICE_USAGE_CONFIG_UPDATE','SMS_SERVICE_CONFIG_UPDATE','SMS_SERVICE_USAGE_CONFIG_UPDATE', 'VERIFICATION_SERVICE_CONFIG_UPDATE'])
+    FROM atlas_bap_dashboard.access_matrix AS T1
```

Comment:
Why do we need `ONBOARDING_DOCUMENT_CONFIG_UPDATE` and others,  that exist only on BPP side?